addons/binding/org.openhab.binding.nibeheatpump/src/main/java/org/openhab/binding/nibeheatpump/internal/message/NibeHeatPumpMessage.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nibeheatpump.internal.message;
+
+import org.openhab.binding.nibeheatpump.internal.NibeHeatPumpException;
+
+/**
+ * The {@link NibeHeatPumpMessage} define interface for Nibe messages.
+ *
+ *
+ * @author Pauli Anttila - Initial contribution
+ */
+public interface NibeHeatPumpMessage {
+
+    /**
+     * Procedure for present class information in string format. Used for
+     * logging purposes.
+     *
+     */
+    @Override
+    String toString();
```

Comment:
Every `Object` already has this method, defining it in an interface is not needed.